src/components/MembersSectionNew/components/MembersCard/membersCard.module.css

```diff
@@ -1,19 +1,11 @@
 .member_card {
-  display: flex;
-  flex-direction: column;
-  flex: 1 250px;
-  gap: 10px;
-  margin: 20px;
-  border: 1px solid var(--border-color);
-  padding: 20px 10px;
-  background: white;
-  cursor: pointer;
-  max-width: 270px;
-  box-shadow: 0 0 15px -7px rgba(0, 0, 0, 0.65);
-  border-radius: 5%;
-  transition: box-shadow 0.38s ease-out;
-  align-items: center;
+  width: 249px;
```

Comment:
can we please convert px into rem and follow RDS design system

Reply:
sure